xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/transactions/PaymentChannelCreate.java

```diff
@@ -50,17 +50,17 @@ static ImmutablePaymentChannelCreate.Builder builder() {
 
   /**
    * Set of {@link TransactionFlags}s for this {@link PaymentChannelCreate}, which only allows the
-   * {@code tfFullyCanonicalSig} flag.
+   * {@code tfFullyCanonicalSig} flag, which is deprecated.
    *
    * <p>The value of the flags cannot be set manually, but exists for JSON serialization/deserialization only and for
    * proper signature computation in rippled.
    *
-   * @return Always {@link TransactionFlags} with {@code tfFullyCanonicalSig} set.
+   * @return Always {@link TransactionFlags#EMPTY}.
    */
   @JsonProperty("Flags")
-  @Value.Default
+  @Value.Derived
```

Comment:
This should have originally been `@Value.Derived`. The pattern we follow for `@Value.Derived` vs `@Value.Default` is that transactions/ledger objects that have specific flags are `@Value.Default` so that developers can set the flags if they want to, but if they don't set them they'll default to something, and transactions/ledger objects that _don't_ have specific flags are `@Value.Derived` so that you can't set them.

Reply:
But you are correct, this would technically be a breaking change, so I reverted in [this commit](https://github.com/XRPLF/xrpl4j/commit/10cb7381e9d6cdfc640dd11e37d36384d58923e1)

Reply:
Now that we have both empty flags and UNSET or zero flags, I wonder if we should make all `flags` fields `@Value.Default`. Functionally there is no difference between zero and empty flags, but for compatibility with other SDKs, maybe we should allow users to set `flags` to `0`?